resources/function_help/json/if

```diff
@@ -4,10 +4,11 @@
   "description": "Tests a condition and returns a different result depending on the conditional check.",
   "arguments": [
 	{"arg":"condition","description":"the condition which should be checked"},
-	{"arg":"result_when_true","description":"the result which will be returned when the condition is True"},
-	{"arg":"result_when_false","description":"the result which will be returned when the condition is False"}
+	{"arg":"result_when_true","description":"the result which will be returned when the condition returns True or not null value"},
```

Comment:
Thanks for your inputs. 
Ok for `evaluates to`. About the not null value, it wasn't just about `null` but also about zero (0) value (which is a `valeur nulle` in french).
Below are what the if function returns
if (1, 'One', 'Not One') --> 'One'
if (1+1=2, 'One', 'Not One') --> 'One'
if (1+1=3, 'One', 'Not One') --> 'Not One'
if ('1', 'One', 'Not One') --> 'One'
if (0, 'One', 'Not One') --> 'Not One'
if ('0', 'One', 'Not One') --> 'Not One'
if ('00', 'One', 'Not One') --> 'One'
if (null, 'One', 'Not One') --> 'Not One'
if (null is null, 'One', 'Not One') --> 'One'
if ('', 'One', 'Not One') --> 'Not One'
if (' ', 'One', 'Not One') --> 'One'

I hope I covered all the situation. So if someone can summarize this in few words...